README.md

```diff
@@ -10,12 +10,11 @@ Please feel free to contribute Issues and pull requests.
 ## Documentation valid for `master` branch
 
 Please check the documentation for the paperclip version you are using:
-https://github.com/kreeti/paperclip/releases
+https://github.com/kreeti/kt-paperclip/releases
 
 ---
 
-[![Build Status](https://secure.travis-ci.org/kreeti/paperclip.svg?branch=master)](http://travis-ci.org/kreeti/paperclip)
-[![Dependency Status](https://gemnasium.com/kreeti/paperclip.svg?travis)](https://gemnasium.com/kreeti/paperclip)
```

Comment:
This service isn't active anymore. But maybe you want to leave this as a placeholder for future migration.